selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java

```diff
@@ -120,7 +120,7 @@ public void shouldDebugCliPhpScriptFromFirstLine() {
     notificationPopup.waitExpectedMessageOnProgressPanelAndClosed("Remote debugger connected");
 
     projectExplorer.openItemByPath(PATH_TO_LIB_PHP);
-    editor.setBreakPointAndWaitActiveState(14);
+    editor.setBreakpoint(14);
```

Comment:
It's important part of test to ensure that breakpoint becomes an active in some time.

Reply:
It becomes active when php debug session starts.
https://github.com/eclipse/che/blob/22f6bb9e0529144bdef81fe057078e1a5d92d0b8/selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/PhpProjectDebuggingTest.java#L127

Reply:
Spec doesn't say that breakpoint must be activated immediately. But if breakpoint location is valid debugger can activate it later

Reply:
Test is aimed to ensure that breakpoint has ever activated. We can check it later, but IMHO we should check it anyway to catch an error with activation of breakpoint in future.

Reply:
Agree, I will check if breakpoint activated after debug sessions is initialized.